src/bin/js/worker_interface.ml

```diff
@@ -185,7 +185,7 @@ type options = {
   debug_cc : bool option;
   debug_combine : bool option;
   debug_constr : bool option;
-  debug_explanations : bool option;
+  debug_explanation : bool option;
```

Comment:
Please justify `correct`. The name of module is `explanation`.

Reply:
> the current name of the debug flag is `explanation` too

I'd argue that this is incorrect as well (although I never noticed since I always write `-d ex`) ;)

What I mean is — there are multiple explanations used by Alt-Ergo that we want to debug; similar to `debug_commands`, `debug_warnings`, `debug_triggers`, `debug_types`, `debug_intervals`, etc. On the other hand the singular flags such as `debug_bitv`, `debug_interpretation`, `debug_matching`, `debug_uf`, etc. refer to a single data-structure / theory / step to debug (ignoring the fact that we duplicate the environment for case-split purposes).

"The correct name" was a bit strong, and I apologize — I don't think it matters too much either way. However I do believe that if it does not matter, it is best to not change whatever is there. Now that the change has been made let's keep it.

Reply:
I don't think it's matter too. When I see a discrepancy in our naming conventions and I am modifying this part of the code, I try to reduce it a bit. I agree to rename `explanation` to `explanations` everywhere (but for the module name of course).

Reply:
In fact, I removed the source from `Explanation` because `--d explanations` has no relation with this modify. It increase the verbosity of explanations. We may add flags like `--verbosity explanations, ...`.